tests/gdb-tests/tests/test_command_procinfo.py

```diff
@@ -28,6 +39,9 @@ def test_command_procinfo(start_binary):
     assert pid in res_list[1]
     assert "127.0.0.1:31337" in result
 
+    # Close netcat
+    os.killpg(os.getpgid(netcat_process.pid), signal.SIGTERM)
```

Comment:
Is there a reason why we don't kill it via the [`Popen.kill` method](https://docs.python.org/3/library/subprocess.html#subprocess.Popen.kill)?

We should either use that method or if it doesn't make sense, we should recommend why we need to do it in a different manner